content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -43,7 +43,7 @@ give full pre-installation instructions for [RHEL](#k3s-rhel) and
 ### RHEL {#k3s-rhel}
 
 K3s recommends that you turn off 
```

Comment:
K3s is an implementation detail. I think this should be a requirement for RDI
RDI installation might fail if firewalld is not turned off (RDI uses containers and needs changes to IPtables)

Reply:
or the more accurate instructions you give later